holoviews/plotting/bokeh/plot.py

```diff
@@ -241,6 +242,21 @@ def current_handles(self):
         return []
 
 
+    def cleanup(self):
```

Comment:
Would be good to have a few more comment and/or a docstring describing what state is being cleaned up here...

Looks like you simply filter out the subscribers so the unreferenced elements can be garbage collected?

Reply:
Agreed, I'll add a comment now.

Reply:
>Looks like you simply filter out the subscribers so the unreferenced elements can be garbage collected?

Btw, elements or containers will almost never be garbage collected without user intervention since they will usually be in the global IPython namespace. All this cleanup has two main reasons:

* Streams should not push updates to plots which are no longer displayed.
* Plot/Widget/Callback instances can be cleaned up once the plot is no longer displayed.

Reply:
You're right...I wasn't thinking carefully enough as subscribers don't point to elements but to plot methods (iirc). So your explanation makes sense and just need to be expressed in the comment...